WikipediaReadingLists.safariextension/Info.plist

```diff
@@ -13,7 +13,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.2</string>
+	<string>0.3</string>
 	<key>CFBundleVersion</key>
 	<string>2</string>
```

Comment:
Sorry, that last comment pertains to this value.  (Not used to reviewing on GitHub...)

Reply:
Done